{"time":"2025-03-17T23:11:14.6115475+05:30","level":"INFO","msg":"stream: starting","core version":"0.19.8","symlink path":"C:\\Users\\praha\\Documents\\IIT madras\\Acads\\sem 10\\introtoDL\\DA6401_A01\\wandb\\run-20250317_231114-aazy2cj0\\logs\\debug-core.log"}
{"time":"2025-03-17T23:11:14.7297496+05:30","level":"INFO","msg":"created new stream","id":"aazy2cj0"}
{"time":"2025-03-17T23:11:14.7300056+05:30","level":"INFO","msg":"stream: started","id":"aazy2cj0"}
{"time":"2025-03-17T23:11:14.7301641+05:30","level":"INFO","msg":"sender: started","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:11:14.7300056+05:30","level":"INFO","msg":"writer: Do: started","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:11:14.7300056+05:30","level":"INFO","msg":"handler: started","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:11:35.7604626+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:11:38.2000009+05:30","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-03-17T23:11:59.9840341+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:12:03.8375891+05:30","level":"INFO","msg":"Stopping system monitor"}
{"time":"2025-03-17T23:12:03.8380983+05:30","level":"INFO","msg":"Stopped system monitor"}
{"time":"2025-03-17T23:12:04.8510531+05:30","level":"INFO","msg":"handler: operation stats","stats":{"operations":[{"desc":"uploading history steps 0-8, summary","runtime_seconds":2.5285532},{"desc":"uploading config.yaml","runtime_seconds":0.0914725,"progress":"1003B/1003B"}],"total_operations":2}}
{"time":"2025-03-17T23:12:05.1425204+05:30","level":"INFO","msg":"fileTransfer: Close: file transfer manager closed"}
{"time":"2025-03-17T23:12:23.3579681+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:12:46.8700899+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:13:05.2959621+05:30","level":"INFO","msg":"handler: operation stats","stats":{"operations":[{"desc":"uploading history steps 0-8, summary","runtime_seconds":62.9734622,"error_status":"retrying: Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}],"total_operations":1}}
{"time":"2025-03-17T23:13:12.0150673+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:13:42.8481509+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:14:05.9631354+05:30","level":"INFO","msg":"handler: operation stats","stats":{"operations":[{"desc":"uploading history steps 9-9, summary","runtime_seconds":3.2631904}],"total_operations":1}}
{"time":"2025-03-17T23:14:23.7382228+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/prahaladvathsan-iit-madras/DA640_A01/aazy2cj0/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-17T23:14:26.1911509+05:30","level":"INFO","msg":"stream: closing","id":"aazy2cj0"}
{"time":"2025-03-17T23:14:26.1911509+05:30","level":"INFO","msg":"handler: closed","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:14:26.191706+05:30","level":"INFO","msg":"writer: Close: closed","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:14:26.191706+05:30","level":"INFO","msg":"sender: closed","stream_id":"aazy2cj0"}
{"time":"2025-03-17T23:14:26.1923026+05:30","level":"INFO","msg":"stream: closed","id":"aazy2cj0"}
